target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr840n-v2.c

```diff
@@ -0,0 +1,127 @@
+/*
+ *  TP-LINK TL-WR841N/ND v9/v11 / TL-WR842N/ND v3
```

Comment:
Please, drop both mach-*.c files and include changes needed by these boards inside mach-tl-wr841n-v9.c, as the code is almost exact copy and it really doesn't make sense to multiply it.

[You can take a look how it was done with TL-WR842N/ND v3 support](https://github.com/lede-project/source/commit/0b45bec22c59cc6e6b2fa71dc9a88386c5587132).